src/HealthChecks.Publisher.CloudWatch/CloudWatchPublisher.cs

```diff
@@ -0,0 +1,133 @@
+using System.Reflection;
+using Amazon;
+using Amazon.CloudWatch;
+using Amazon.CloudWatch.Model;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace HealthChecks.Publisher.CloudWatch;
+
+/// <summary>
+/// A health check publisher for AWS CloudWatch.
+/// </summary>
+internal class CloudWatchPublisher : IHealthCheckPublisher, IDisposable
```

Comment:
Also are you sure you want this class to be internal?

Reply:
yes

Reply:
`SeqPublisher` is public. I'm fine either way.